src/convert.rs

```diff
@@ -0,0 +1,42 @@
+pub trait CAPIConverter {
```

Comment:
If I may quibble with the trait name, the types implementing the trait are convertees, not converters :)

Reply:
The two hardest things in computer science: Naming things, cache invalidation, and off by one errors.

Maybe `CAPIConvertible`?

Reply:
Works for me